pkg/metrics/metrics.go

```diff
@@ -719,9 +745,10 @@ func init() {
 	prometheus.MustRegister(clusterStatusCountMetric)
 	prometheus.MustRegister(kafkaPerClusterCountMetric)
 	prometheus.MustRegister(clusterStatusCapacityMaxMetric)
-	prometheus.MustRegister(KasFleetManagerClusterProviderResourceQuotaConsumedMetric)
 	prometheus.MustRegister(clusterStatusCapacityUsedMetric)
 	prometheus.MustRegister(clusterStatusCapacityAvailableMetric)
+	prometheus.MustRegister(ClusterProviderResourceQuotaConsumedMetric)
+	prometheus.MustRegister(clusterProviderResourceQuotaMaxAllowedMetric)
```

Comment:
Is there some reason one of the variables is exported and the other one is unexported?

Reply:
ah good catch, didn't notice that. No reason but it should be unexported to keep consistent with the rest of the metrics defined here.